### ISSUE TITLE
Exploit: Double hit from projectile

### ISSUE THREAD

**Issue by Craluminum2413** [anegostudios/VintageStory-Issues#811]
**Game Version:** 1.14.8, 1.15.0-pre.12
**Platform:** Windows 10
**Modded:** No

### How to reproduce
1. `Throw a spear`/`shoot a bow` from above right at the entity
2. Spear/arrow hit twice

### Videos

https://user-images.githubusercontent.com/69315569/121820258-91ea1880-cc9a-11eb-94b2-a83f0e8b5499.mp4

https://user-images.githubusercontent.com/69315569/121820259-931b4580-cc9a-11eb-9655-9ca3133b9ff5.mp4

**Comment by BinaryKali** [comment]
I've run into this one as well. I've even had it hit more than twice sometimes.

**Comment by Craluminum2413** [comment]
@BinaryKali What versions of the game do you mean?

**Comment by BinaryKali** [comment]
I'm currently in 1.15-pre2 and can definitely say it's happened to me there. I had the double hit happen just last night a couple of different times. I want to say that I've had it happen in 1.14.10 as well but can't say 100% for sure as I've only been playing the prerelease recently.

**Comment by Craluminum2413** [comment]
I forgot to add new versions to the description, I added them now

**Comment by Craluminum2413** [comment]
Also happens with arrows

**Comment by Craluminum2413** [comment]
What are the priorities for exploits?

**Comment by copygirl** [comment]
> What are the priorities for exploits?

Depends on the severity of the exploit. They're considered more severe if the exploit affects other players in some way, such as finding and abusing a way to intentionally put a multiplayer server into a crash loop, or something that can affect a survival server's economy.

**Comment by Craluminum2413** [comment]
Still can hit myself while hunting because of this bug

**Comment by copygirl** [comment]
Confirmed to still happen in a 1.16.4 release candidate.

**Comment by Craluminum2413** [comment]
Fixed in 1.12.0-pre.6
```
Fixed: Should fix spears damaging creatures multiple times on impact
```

**Comment by Galdor-mithr** [comment]
yes like this
